File: content/api-documentation/how-to/assets/examples/listAssets.go
package main

import (
	"github.com/financefeasthq/financefeast-trade-api-go/financefeast"
)

func init() {
	financefeast.SetBaseUrl("https://paper-api.financefeast.markets")
}

func main() {
	// Get a list of all active assets.
	status := "active"
	assets, err := financefeast.ListAssets(&status)
	if err != nil {
		panic(err)
	}

	// Filter the assets down to just those on NASDAQ.
	nasdaq_assets := []financefeast.Asset{}
	for _, asset := range assets {
		if asset.Exchange == "NASDAQ" {
			nasdaq_assets = append(nasdaq_assets, asset)
		}
	}
}

File: content/api-documentation/how-to/assets/examples/listAssets.js
const financefeast = require('@financefeasthq/financefeast-trade-api')
const financefeast = new financefeast()

// Get a list of all active assets.
const activeAssets = financefeast.getAssets({
    status: 'active'
}).then((activeAssets) => {
    // Filter the assets down to just those on NASDAQ.
    const nasdaqAssets = activeAssets.filter(asset => asset.exchange == 'NASDAQ')
    console.log(nasdaqAssets)
})
